edscrapers/scrapers/base/helpers.py

```diff
@@ -4,8 +4,11 @@
 import datetime
 import hashlib
 import urllib.parse
+import itertools
 import requests
 import bs4
+import ratelimit
```

Comment:
I wonder if there's any point extracting this into more generic helpers, so they'd be available for other parsers as well... :thinking: